Modules/_ssl.c

```diff
@@ -137,6 +138,8 @@ struct py_ssl_library_code {
 #else /* OpenSSL < 1.1.0 */
 #if defined(WITH_THREAD)
 #define HAVE_OPENSSL_CRYPTO_LOCK
+/* For some reason, this function is not declared on OpenSSL's headers */
+void OPENSSL_cpuid_setup(void);
```

Comment:
No it was not. I just noticed it is inside of this definition. Great catch!
It is intended to be defined only when `/* OpenSSL < 1.1.0 */`.